components/src/status_im/ui/components/react.cljs

```diff
@@ -92,11 +92,17 @@
     (vec (map string/upper-case ts))
     ts))
 
+(defn escape-html [text]
```

Comment:
that would also apply on mobile is that desirable ?

Reply:
oh wait, no it's not

Reply:
fixed

Reply:
Looks good for me. I'll add issue in `react-native-desktop`.

Reply:
https://github.com/status-im/react-native-desktop/issues/349